examples/all-clusters-app/all-clusters-common/gen/IMClusterCommandHandler.cpp

```diff
@@ -4903,7 +4903,7 @@ void DispatchServerCommand(app::CommandHandler * apCommandObj, CommandId aComman
 
 namespace NetworkCommissioning {
 
-void DispatchServerCommand(app::CommandHandler * apCommandObj, CommandId aCommandId, EndpointId aEndpointId,
+void __attribute__((weak)) DispatchServerCommand(app::CommandHandler * apCommandObj, CommandId aCommandId, EndpointId aEndpointId,
```

Comment:
Will we consider remove the use of DispatchServerCommand and use IM->RegisterServerCommandDispatcher (or other similar functions?)

Reply:
If we go down the path of dynamic handler registration, then yes, it will remove the "DispatchServerCommand" approach.